### ISSUE TITLE
Option for full opacity preview images

### ISSUE THREAD

**Issue by deciare** [drawthingsai/draw-things-community#9]
In version 1.20240805.1, when generating images, the low-resolution preview starts out almost transparent, gradually fading in over the course of the generation.

That makes it difficult to see the preview in the early steps of the generation. When using large models like FLUX.1 [dev], generation takes long enough that it is useful to cancel the generation if early steps don't look promising. It is difficult to do that if I can't see what's in the preview until the generation is almost complete.

I think it would be useful to have an option that causes the preview to be fully opaque, and therefore easily visible, as soon as a preview image is available.

**Comment by czkoko** [comment]
Yes, the translucent preview hidden under the progress bar does affect the user experience. 
In addition, I think Tiny VAE can be used to get a higher-definition preview, and the impact on performance is almost negligible. SD1.5, SDXL, SD3, Flux have been supported
https://huggingface.co/madebyollin
